client/src/components/grants/inputs.tsx

```diff
@@ -187,8 +187,8 @@ export function Select({
         {label}
       </label>
       {required && (
-        <span className="absolute text-purple-700 inset-y-0 right-0">
-          * Required
```

Comment:
I've been adding spaces there, are we not supposed to? I can remove them if that's the case

Reply:
I was trying to follow the Figma and looked like there was no space.